AdaptiveRoads/UI/QuayRoads/UIDraggableWindow.cs

```diff
@@ -0,0 +1,117 @@
+using ColossalFramework;
+using ColossalFramework.UI;
+using KianCommons;
+using KianCommons.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+using static KianCommons.ReflectionHelpers;
+
+// TODO: move to KianCommons; proper close button
```

Comment:
- move this to KianCommons?
TODO:
- proper close button
- resizing
- scrollbars

Reply:
this does not work very well with drop down does it?

also in TODO:
- fit to screen when resolution changes the way it is done in UUI.
but probabely put the todos in the code so

Reply:
it does work fine with drop down afaik.